example/simple/Dockerfile

```diff
@@ -0,0 +1,16 @@
+
+FROM busybox
+
+RUN echo 1 > /a
+RUN echo 2 > /b
+VOLUME /bar
+RUN mkdir /c && echo 3 > /c/a && echo 4 > /c/b
+RUN rm /bin/ls
+RUN rm /b
+EXPOSE 8080
+RUN rm /c/b
+VOLUME /baz
+EXPOSE 3000
+# RUN echo > /bin/hostname
+RUN rm /bin/ping
+# RUN rm /bin/hostname
```

Comment:
Yep that's a good shout, that case is covered [in this test case](https://github.com/duedil-ltd/portainer/pull/39/files#diff-e86caccab63ebed4fa7dd443439819e9R154). We first wrote a, then we deleted it, and then added it again.